Capstone_EDA.ipynb

```diff
@@ -59,38 +59,38 @@
     "## 2.3 Feature Glossary\n",
```

Comment:
<p>Im nächsten Schritt sollten für die ganzen Codezellen in Abschnitt 5 und 6 unbedingt noch Schlussfolgerungen hinzukommen: Warum haben wir diese Funktionen ausgeführt, was sagen sie uns? So stehen sie gerade noch etwas erkenntnislos für sich.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/davidslabon/Capstone_UnlockingClimateSolutions/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='79'/>

Reply:
<p>Ja klar, das ist auch alles andere als fertig.</p>